components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -475,6 +627,10 @@ def main(argv):
             start_results_cache(instance_id, clp_config, conf_dir)
         if '' == component_name or SCHEDULER_COMPONENT_NAME == component_name:
             start_scheduler(instance_id, clp_config, container_clp_config, mounts)
+        if '' == component_name or SEARCH_SCHEDULER_COMPONENT_NAME == component_name:
```

Comment:
Also need to add `SEARCH_SCHEDULER_COMPONENT_NAME` under `Validate and load necessary credentials`